src/app/public/modules/autocomplete/autocomplete.component.spec.ts

```diff
@@ -18,17 +18,19 @@ import { SkyAutocompleteComponent } from './autocomplete.component';
 import { SkyAutocompleteInputDirective } from './autocomplete-input.directive';
```

Comment:
Should we clean up these imports while we're here.

Reply:
done.